tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAssetRepository.scala

```diff
@@ -35,6 +35,8 @@ trait PublicAssetRepository {
     SMono(get(username, id))
       .map(publicAsset => (publicAsset.identityIds.toSet ++ identityIdsToAdd.toSet) -- identityIdsToRemove.toSet)
       .flatMap(identityIds => SMono(update(username, id, identityIds)))
+
+  def getTotalSize(username: Username): Publisher[Long]
```

Comment:
Should we add unit tests for this method?
in `PublicAssetRepositoryContract` 
At least case: "shouldReturnZeroWhenDefault"